src/renderer/features/wallet-select/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { GROUP_LABELS, WalletGroup } from './components/WalletGroup';
+import { walletsSelectFeatureStatus } from './model/feature';
+import { walletSelectModel } from './model/wallet-select-model';
+import { walletSelectService } from './service/walletSelectService';
+
+export const walletSelectFeature = {
+  feature: walletsSelectFeatureStatus,
+  walletSelectService,
```

Comment:
Maybe put this kind of variables into `services` property?
Or it will go away in future with more detailed refactoring?